systems/plants/@RigidBodyManipulator/doKinematics.m

```diff
@@ -254,15 +254,15 @@
   kinsol.J = computeJ(kinsol.T, S);
   
   if options.compute_gradients
-    [kinsol.qdotToV, kinsol.dqdotToVdq] = qdotToV(model, q);
-    [kinsol.vToqdot, kinsol.dvToqdotdq] = vToqdot(model, q);
+    [kinsol.qdotToV, kinsol.dqdotToVdq] = computeQdotToV(bodies, q);
+    [kinsol.vToqdot, kinsol.dvToqdotdq] = computeVToqdot(bodies, q);
   else
-    kinsol.qdotToV = qdotToV(model, q);
-    kinsol.vToqdot = vToqdot(model, q);
+    kinsol.qdotToV = computeQdotToV(bodies, q);
+    kinsol.vToqdot = computeVToqdot(bodies, q);
```

Comment:
Decided to change kinsol.qdotToV and kinsol.vToQdot to a cell array of matrices instead of one big matrix.